mobile/src/people/employees-list-item.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { TouchableHighlight, StyleSheet, Platform, Text, View, Image } from 'react-native';
+
+import { Avatar } from '../people/avatar';
+import { Employee } from '../reducers/organization/employee.model';
+import { employeesListItemStyles as styles } from './styles';
+
+interface EmployeesListItemProps {
+    employee: Employee;
+    id: string;
+}
+
+export class EmployeesListItem extends React.Component<EmployeesListItemProps> {
+    public render() {
+        const employeeName = this.props.employee ? this.props.employee.name.replace(',', '') : 'Unknown';
```

Comment:
please remove `replace`. I think we'll have to define it one place (Employee class, I'd say), we'll do it later.

Reply:
ok